### ISSUE TITLE
Move examples to tutorials repo?

### ISSUE THREAD

**Issue by mkelley** [NASA-Planetary-Science/sbpy#290]
Here, we have an example directory with a couple notebooks, but we also have documentation that could include examples, and a separate sbpy-tutorial repository.  I didn't even remember that the examples directory existed.  Can we move examples to the tutorials and drop this directory?

**Comment by jianyangli** [comment]
I agree.  It's more pythonic to integrate examples into documentations.  For more complicated examples, the tutorial repository is a good place to go.

**Comment by jianyangli** [comment]
I just checked what's in the example directory.  Other than two .rst files, there are three ipython notebooks files:

estimating-coma-continuum.ipynb
- Contains one example, which makes use of @mkelley 's proprietary `mskpy` library.
- I think this file is significantly outdated and can be dropped.  The content should have been covered by dust-coma-brightness.ipynb in sbpy-tutorial repository.

haser-model.ipynb
- Contains two examples, one to reproduce Fig. 1 of Combi et al. (2004), and the other to reproduce Newburn & Johnson (1978) CN, C2, and C3 production rates.
- The notebook runs well.
- I think the examples here are good to be merged into haser-model.ipynb in sbpy-tutorial repo.

vectorial-model.ipynb
- Lots of content in it, and runs well on my computer.
- This file appears to be pretty new.
- I think this one can be simply moved to sbpy-tutorial in the activity directory.

If all sounds good to you, @mkelley and @migueldvb , I can implement what I suggested above, at least get things started so the `sbpy` repo can be cleaned up for v0.3 release.  Please let me know if you want me to proceed this way, or if you have other suggestions.

**Comment by mkelley** [comment]
@jianyangli Your plan sound good, thanks.
